buildspec_load_test.yml

```diff
@@ -52,21 +52,13 @@ phases:
       - python -m venv venv
       - source venv/bin/activate
       - pip install -r ./load_tests/requirements.txt
-      # AWS configure sts temp credentials
```

Comment:
You may wonder, why did he remove this? Is this safe?

In load tests, everything runs via the python script, and as you can see in the other code changes, that code assumes the `LOAD_TEST_CFN_ROLE_ARN` role itself. This means we were previously unnecessarily chaining sessions (buildspec assumes the role, then the python uses that role to assume the same role).